src/content/docs/validator/external-chains/linea.mdx

```diff
@@ -233,12 +233,14 @@ vald's `config.toml`
 ```bash
 [[axelar_bridge_evm]] name = "linea" rpc_addr = "http://IP:PORT"
 start-with-bridge = true
+finality_override = "confirmation"
 ```
   </tab-item>
   <tab-item title="Testnet" class="bleed">
 ```bash 
 [[axelar_bridge_evm]] name = "linea" rpc_addr = "http://IP:PORT"
```

Comment:
Name is outdated, should be linea-sepolia. Can you sanity check other chains (since we added many sepolia ones) as well?

Reply:
Will do under separate pr